sdk/aqueduct/integrations/connect_config.py

```diff
@@ -43,6 +43,9 @@ class PostgresConfig(BaseConnectionConfig):
     host: str
     port: Optional[str] = "5432"
 
+class RedshiftConfig(PostgresConfig):
+    port: Optional[str] = "5439"
```

Comment:
Where does this default port number come from? Can we document it here and in `test-config-example.yml` too? Eg. it would be good to know, if we had a script like `start_redshift.py`, whether the port argument was required or not.

Reply:
Added